plugins/legacy-jbrowse/src/JBrowse1TextSeachAdapter/JBrowse1TextSearchAdater.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Small typo in the filename: adater -> adapter

Reply:
thanks for catching that!